### PR TITLE
Release 0.2.1

### PR DESCRIPTION
**Pull request by dalenewby**
###Added
* Variable `aws_region`

###Changed
* Log retention increased to 60 days

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -8,7 +8,7 @@ locals {
  */
 resource "aws_cloudwatch_log_group" "cw_b2_fsbackup" {
   name              = local.app_name_and_env
-  retention_in_days = 45
+  retention_in_days = 60
```
Comment (briskt) [inline main.tf:11]: Is this the new recommendation? Just wondering what I should use the next time I go there.

File: variables.tf [hunk 2]
```diff
@@ -13,6 +13,12 @@ variable "app_name" {
   type = string
 }
 
+variable "aws_region" {
+  description = "AWS region to run in (e.g., us-east-1)"
+  default     = ""
```
Comment (briskt) [inline variables.tf:18]: Why not `"us-east-1"`? I _think_ having this as the default could cause an error if it's not specified.

**Review by briskt — APPROVED** [review]
Couple questions for you

File: main.tf (hunk 1 above)
Comment (dalenewby) [reply main.tf:11]: Not necessarily.  You said someone wanted 45 days.  45 isn't valid but 60 is so it meets (and exceeds) the "requirement" for 45 days.

File: variables.tf (hunk 2 above)
Comment (dalenewby) [reply variables.tf:18]: I think I want it to error if it isn't specified.  The user of the module should specify what region he wants it to run in.

Comment (briskt) [reply variables.tf:18]: Ok, your preference, but personally I would remove the default for that.

**Pull request merged**